include/cell.h

```diff
@@ -249,8 +277,17 @@ class Cell {
 
   //! Shape function
   std::shared_ptr<const Element<Tdim>> element_{nullptr};
+
+  //! Velocity constraints
+  //! first-> face_id, second->pair of direction [0/1/2] and velocity value
+  std::vector<std::tuple<unsigned, unsigned, double>> velocity_constraints_;
```

Comment:
this should be a map

Reply:
Not sure actually we can do this...

Because the first `unsigned` is face id, and for a face id, there can be multiple sets of direction and velocity. Thus it might not work when we call them? What do you think?

Reply:
Can we not have a map of a pair?